docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -1,34 +1,30 @@
 [id="migrating-databases-to-mariadb-instances_{context}"]
 
-//kgilliga: This module will be converted to a procedure. Unclear which assembly it should be included in.
 //Check xref contexts.
+//kgilliga: Find out if the steps in the Variables and pre-checks sections can go in the main procedure or if they have to be done before.
 
 = Migrating databases to MariaDB instances
 
 This document describes how to move the databases from the original
-OpenStack deployment to the MariaDB instances in the OpenShift
-cluster.
+{osp_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
 
-____
-*NOTE* This example scenario describes a simple single-cell setup. Real
+[NOTE] 
+This example scenario describes a simple single-cell setup. Real
 multi-stack topology recommended for production use results in different
 cells DBs layout, and should be using different naming schemes (not covered
 here this time).
-____
 
-== Prerequisites
+.Prerequisites
 
 * Make sure the previous Adoption steps have been performed successfully.
- ** The OpenStackControlPlane resource must be already created at this point.
- ** Podified MariaDB and RabbitMQ are running. No other podified
-control plane services are running.
- ** Required services specific topology. For more information, see xref:pulling-the-openstack-configuration_{context}[Pulling the OpenStack configuration].
+ ** The `OpenStackControlPlane` resource must be already created.
+ ** The control plane MariaDB and RabbitMQ are running. No other control plane services are running.
+ ** Required services specific topology. For more information, see xref:pulling-the-openstack-configuration_{context}[Pulling the {osp_prev_long} configuration].
```

Comment:
This link doesn't work in the preview

Reply:
I will update in a separate PR when I eventually rename the "Pulling configuration" module.